README.md

```diff
@@ -83,7 +83,7 @@ If you want to implement your own source, it must match [RegistrySource interfac
 | CL_API_GET_BLOCK_INFO_MAX_RETRIES                | Ethereum consensus layer max retries for fetching block info. Independent of `CL_API_MAX_RETRIES`                                          | false        | 1                                       |
 | FETCH_INTERVAL_SLOTS                             | Count of slots in Ethereum consensus layer epoch                                                                                           | false        | 32                                      |
 | CHAIN_SLOT_TIME_SECONDS                          | Ethereum consensus layer time slot size                                                                                                    | false        | 12                                      |
-| START_SLOT                                       | Ethereum consensus layer slot for start Balval                                                                                             | false        | 1518000                                 |
+| START_EPOCH                                      | Ethereum consensus layer epoch for start Balval                                                                                            | false        | 1518000                                 |
```

Comment:
The default value differs from one in .env.example

Reply:
fixed